test/index.js

```diff
@@ -0,0 +1,65 @@
+var assert = require('assert')
+var Trace = require('..')
+
+suite('URL Shorteners', function () {
+  this.timeout(10000)
+
+  suite.skip('Twitter (t.co)', function () {
```

Comment:
Yes, I haven't fixed the weird Twitter redirects in this PR, because you started a branch which fixes that, if I remember correctly. I figured it'll make more sense to consolidate them instead of doing the work twice :)